modules/jt-examples-of-common-use-cases.adoc

```diff
@@ -0,0 +1,172 @@
+        - name: maven-settings
+          workspace: maven-settings
+      params:
+        - name: CONTEXT_DIR
+          value: "$(params.context-path)"
+        - name: GOALS
+          value: ["test"]
+    - name: mvn-package
+      taskRef:
+        name: maven
+      runAfter:
+        - mvn-tests
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+        - name: maven-settings
+          workspace: maven-settings
+      params:
+        - name: CONTEXT_DIR
+          value: "$(params.context-path)"
+        - name: GOALS
+          value: ["package"]
+    - name: create-image-and-deploy
+      taskRef:
+        name: openshift-client
+      runAfter:
+        - mvn-package
+      workspaces:
+        - name: manifest-dir
+          workspace: shared-workspace
+        - name: kubeconfig-dir
+          workspace: kubeconfig-dir
+      params:
+        - name: SCRIPT
+          value: |
+            cd "$(params.context-path)"
+            mkdir -p ./artifacts/deployments && cp ./target/*.war ./artifacts/deployments
+            oc new-project helloworldproject
+            oc project helloworldproject
+            oc process -f jboss-eap70-binary-build.json | oc create -f -
+            oc start-build eap-helloworld-app --from-dir=artifacts/
+            oc new-app jboss-eap70-deploy.json
+
+----
+
+== Extending the core capabilities of Jenkins and Tekton by using plugins
+Jenkins has the advantage of a very rich ecosystem of numerous plugins developed over the years by its extensive user base. You can search and browse the plugins in the link:https://plugins.jenkins.io/[Jenkins Plugin Index]. 
+
+The comparatively newer Tekton also has many tasks developed and contributed by the community and enterprise users. A publicly available catalogue of reusable Tekton tasks are available in the link:https://hub.tekton.dev/[Tekton Hub].
+
+In addition, Tekton incorporates many of the plugins of the Jenkins ecosystem within its core capabilities. For example, authorization is a critical function in both Jenkins and Tekton. While Jenkins ensures authorization using the link:https://plugins.jenkins.io/role-strategy/[Role-based Authorization Strategy] plugin, Tekton has built-in xref:../../authentication/using-rbac.adoc#using-rbac[Role-based Access Control] (RBAC).
+
+== Sharing reusable code in Jenkins and Tekton
+Jenkins link:https://www.jenkins.io/doc/book/pipeline/shared-libraries/[shared libraries] provide reusable code for parts of Jenkins pipelines. The libraries are shared between link:https://www.jenkins.io/doc/book/pipeline/jenkinsfile/[Jenkinsfiles] to create highly modular pipelines without code repetition.
+
+Although there is no direct equivalent of Jenkins shared libraries in Tekton, you can achieve similar workflows by using tasks from the link:https://hub.tekton.dev/[Tekton Hub], in combination with xref:./jt-extending-tekton-capabilities-using-custom-tasks-and-scripts.adoc[custom tasks and scripts].
```

Comment:
./jt-extending-tekton-capabilities-using-custom-tasks-and-scripts.adoc this link seems to be 404 here https://deploy-preview-42629--osdocs.netlify.app/openshift-enterprise/latest/cicd/jenkins-tekton/migrating-from-jenkins-to-tekton#jt-examples-of-common-use-cases_migrating-from-jenkins-to-tekton

Reply:
oh is it because it is referring to our existing docs